src/redux/actions/offeringActions.js

```diff
@@ -1,8 +1,9 @@
 import { MAKE_OFFER } from "../constants/actionTypes";
 
-export const makeOffer = (offering) => {
+export const makeOffer = (offering, postId) => {
```

Comment:
needed postId to be easily update the current posting in `postings`  in the redux store